packages/design/jobberStyle.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require("fs");
+// eslint-disable-next-line import/no-internal-modules
+const customPropertiesObject = require("./src/foundation.js");
+
+const regexExpressions = {
+  cssVars: /var\((.*)\)/,
+  calculations: /calc\((.*)\)/,
+  removeAllNonNumerals: /[^0-9.+\-/*]/gi,
+  extractAllVarGroups: /var\(.*?\)/g,
+};
+
+const customProperties = customPropertiesObject.customProperties;
+
+const resolvedCssVars = getResolvedCSSVars(customProperties);
+
+const jsonContent =
+  "export const JobberStyle = " + JSON.stringify(resolvedCssVars, undefined, 2);
+
+fs.writeFile("./foundation.js", jsonContent, "utf8", function(err) {
+  if (err) {
+    console.log("An error occured while writing JSON object to File.");
+    return console.log(err);
+  }
+  console.log("JSON file has been saved.");
+});
+
+//jobberStyle recursively resolves css vars
+//varRegexResult returns --base-unit from var(--base-unit)
+//calcRegexResult returns var(--base-unit) / 16 from calc(var(--base-unit) / 16)
```

Comment:
Might be clearer to reference these inline in the method near where they are relevant.